src/sentry/workflow_engine/migrations/0055_workflowengine_rename_column.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.1.7 on 2025-05-06 00:29
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("workflow_engine", "0054_clean_up_orphaned_metric_alert_objects"),
+    ]
+
+    operations = [
```

Comment:
I _think_ you need to do a `SeparateDatabaseAndState` here similar to [this](https://github.com/getsentry/sentry/blob/0774351fdaeaa015b10a49ae9d60fe0fcad7d291/src/sentry/replays/migrations/0002_rename_to_segment_id.py#L31) example I found. Bonus points if you flesh out [this section](https://develop.sentry.dev/backend/application-domains/database-migrations/#renaming-columns) of the dev docs cause it's not as comprehensive as it could be.